pkg/server/workflow.go

```diff
@@ -0,0 +1,257 @@
+package server
+
+import (
+	"context"
+	"encoding/csv"
+	"encoding/json"
+	"errors"
+	"fmt"
+	guuid "github.com/google/uuid"
+	api "github.com/pennsieve/pennsieve-agent/api/v1"
+	pb "github.com/pennsieve/pennsieve-agent/api/v1"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/viper"
+	"golang.org/x/exp/slices"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strconv"
+	"strings"
+)
+
+type WorkOrder struct {
+	ProcessID       guuid.UUID                       `json:"ProcessID"`
+	FilePath        string                           `json:"FilePath"`
+	ManifestId      int32                            `json:"ManifestId"`
+	WorkFlowType    pb.WorkflowResponse_WorkflowType `json:"WorkFlowType"`
+	WorkOrderInput  string                           `json:"WorkOrderInput"`
+	WorkOrderFiles  string                           `json:"WorkOrderFiles"`
+	WorkOrderStatus bool                             `json:"WorkOrderStatus"`
```

Comment:
minor: field names could be: Type,Input,Output; since the struct is already WorkOrder.